src/Auth/project.json

```diff
@@ -16,7 +16,8 @@
     "Npgsql.EntityFrameworkCore.PostgreSQL": "1.0.0-rc2-release1",
     "Npgsql.EntityFrameworkCore.PostgreSQL.Design": "1.0.0-rc2-release1",
     "Npgsql": "3.1.1",
-    "OpenIddict": "1.0.0-*",
+    "OpenIddict.Core": "1.0.0-alpha2-0288",
```

Comment:
@PinpointTownes : Thanks for the information and I corrected the same. When I referred to the latest `"OpenIddict": "1.0.0-alpha2-0218"` i was getting an error but using `1.0.0-*` solves.
Could you also share the roadmap of `OpenIddict` so that if would be helpful for us?